mirgecom/simutil.py

```diff
@@ -434,11 +434,20 @@ def create_parallel_grid(comm, generate_grid):
     return generate_and_distribute_mesh(comm=comm, generate_mesh=generate_grid)
 
 
+def configurate(config_key, config_object=None, default_value=None):
+    """Return a configured item from a configuration object."""
+    if config_object is not None:
+        if config_key in config_object:
+            return config_object[config_key]
```

Comment:
```suggestion
def configurate(config_key, config_object=None, default_value=None):
    """Return a configured item from a configuration object."""
    if config_object is not None:
        if isinstance(config_object, dict):
            d = config_object
        else:
            d = config_object.__dict__
        if config_key in d:
            return d[config_key]
```

Reply:
`d = config_object if isinstance(config_object, dict) else config_object.__dict__`?

Reply:
```violation = any([max_field_errors[i] > configurate(fieldname, field_tol,
TypeError: '>' not supported between instances of 'int' and 'NoneType'
```
Those changes cause this typeerror.

Reply:
What did you provide as the `tolerance` argument to `configurate`?

Reply:
The function argument `tolerance`

Reply:
Yes, that return is necessary

Reply:
Just make sure to put it back to the function level as it was before.  See the [comment below](https://github.com/illinois-ceesd/mirgecom/pull/716#discussion_r927839180).